### ISSUE TITLE
Prompt before starting break?

### ISSUE THREAD

**Issue by alphapapa** [lolownia/org-pomodoro#45]
I would really like to use org-pomodoro, because I've used the technique successfully with a shell script I wrote.  But there is one problem: When I'm engaged in a task, and the timer runs out, I may be "in the zone" and may want to get to a "stopping point" before I start a break.  But if org-pomodoro automatically pauses the task and starts the break period, this forces me to take a break early, fix the timer, or lose track of my time.

Could you please add an option to prompt for the break period instead of starting it automatically?  It would be great if it waited for me to execute a command, like maybe `org-pomodoro-start-break-period`, and then I could either bind it to a key or execute it directly with `M-x`.  Also, it would be handy if there were some kind of `org-pomodoro-next-action` or `org-pomodoro-dwim` command that would do the next thing in the sequence `start work->start break->start work`.

Thanks for your work on this handy tool!

**Comment by leoc** [comment]
This is an interesting scenario. Though this undermines the pomodoro technique, I'm sometimes in the same position when I want to do at least 25 minutes on a certain subject.

Currently I am quite busy, so this may take me a while.

**Comment by alphapapa** [comment]
Yeah, for me, I'm not usually in a position to just jump out of my chair as soon as the 25 minutes ends.  :)

Thanks.  I'm a lisp novice, but if I get some time I'll see what I can do.

**Comment by alecigne** [comment]
@alphapapa Hi, could you make any progress on this matter? I am also interested.

**Comment by alphapapa** [comment]
@alecigne Hi, well, I'm not exactly a lisp novice anymore (though still on that end of the spectrum), so I might be able to do something now.  No promises on what or when, but I'll put it on my Org Agenda.  ;)  Thanks for the reminder.

**Comment by alecigne** [comment]
@alphapapa Thank you for your answer (and for pushing the task into your Org agenda). I'd cook something myself but I'm still a Lisp novice, I'm afraid 😄

**Comment by azzamsa** [comment]
@alphapapa need more reminder ? :)

**Comment by chrisbodhi** [comment]
Does PR #77 solve this use case?

"Add functionality to stop a running pomodoro timer only on demand."
